connectors/services/job_scheduling.py

```diff
@@ -167,13 +177,19 @@ async def _run(self):
                         native_service_types=native_service_types,
                         connector_ids=connector_ids,
                     ):
-                        await self._schedule(connector)
+                        if not self.schedule_tasks_pool.try_put(
+                            functools.partial(self._schedule, connector)
+                        ):
+                            connector.log_debug(
+                                f"{self.display_name.capitalize()} service is already running {self.max_concurrency} sync jobs and can't run more at this poinit. Increase '{self.max_concurrency_config}' in config if you want the service to run more sync jobs."  # pyright: ignore
```

Comment:
```suggestion
                                f"{self.display_name.capitalize()} service is already running {self.max_concurrency} concurrent sync jobs and can't run more at this point. Increase '{self.max_concurrency_config}' in config if you want the service to run more concurrent sync jobs."  # pyright: ignore
```

Reply:
Fix typo + I thought adding "concurrent" will make it clear to the user that the sync jobs are running at the same time

Reply:
Good one, I also noticed that the message is wrong and mentions sync jobs, where it should be scheduling jobs. Fixed!